cluster/addons/cluster-monitoring/googleinfluxdb/heapster-controller-combined.yaml

```diff
@@ -32,7 +32,8 @@ spec:
             - --sink=gcl
             - --sink=gcmautoscaling
```

Comment:
Do these tests run for a day? I have seen quota issues only after 16+
hours.

On Mon, Sep 28, 2015 at 11:41 AM, Piotr Szczesniak <notifications@github.com
> wrote:

> In
> cluster/addons/cluster-monitoring/googleinfluxdb/heapster-controller-combined.yaml
> <https://github.com/kubernetes/kubernetes/pull/14559#discussion_r40588758>
> :
>
> > @@ -32,7 +32,8 @@ spec:
> >              - --sink=gcl
> >              - --sink=gcmautoscaling
>
> Will do it. Alternatively we use googleinfluxdb monitoring level in our
> autoscaling e2e tests.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/14559/files#r40588758>.
>